files/cluster_instance_setup.py

```diff
@@ -277,37 +277,41 @@ def panic(title, text):
 
 
 def attach_ebs_volumes(volumes):
-    for volume in volumes:
-        try:
-            metadata = fetch_instance_metadata()
-            boto_ec2_client = boto3.session.Session(
-                region_name=metadata.get("region")
-            ).client("ec2")
-            volume_id, mount_point, device, local_device, detach, create_fs =\
-                volume_values(volume)
-            if check_volume_attachment(
-                boto_ec2_client, volume_id, device, local_device, detach
+    if len(volumes) > 1:
+        LOGGER.info(
```

Comment:
Should this be only `info`? not `warn` of even `error` or maybe throw an exception?

Reply:
Changes to WARM